platform/android/MapboxGLAndroidSDK/proguard-rules.pro

```diff
@@ -2,9 +2,11 @@
 # in ../sdk/tools/proguard/proguard-android.txt,
 # contents of this file will be appended into proguard-android.txt
 -keepattributes Signature, *Annotation*, EnclosingMethod
+-dontobfuscate
```

Comment:
How does this effect our users? Does this mean only the aar file doesn't get obfuscated or are we disabling obfuscating for our users as well?